### PR TITLE
fix: compiling cloud function mangles terminal output

### PR DESCRIPTION
**Pull request by Chriscbr**
Adds a workaround for an issue where compiling cloud functions (which calls esbuild) results in mangled terminal output.

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: libs/wingc/src/jsify.rs [hunk 1]
```diff
@@ -325,15 +325,14 @@ impl<'a> JSifier<'a> {
 					None
 				};
 
-				if !is_resource || fqn.is_none() {
-					format!("new {}({})", ctor, args)
-				} else {
-					let fqn = fqn.expect("expecting fqn to be defined");
+				if let (true, Some(fqn)) = (is_resource, fqn) {
```
Comment (yoav-steinberg) [inline libs/wingc/src/jsify.rs:328]: Is this related to this PR? What's the fix here?

**Review by yoav-steinberg — APPROVED** [review]
This seems fine, can you explain in the top comment the details of the workaround?

File: libs/wingc/src/jsify.rs (hunk 1 above)
Comment (Chriscbr) [reply libs/wingc/src/jsify.rs:328]: Just a refactoring. The way it's currently written, the second branch has to call "expect" to unwrap the FQN even though logically we _know_ the fqn is a `Some(String)`.

**Review by eladb — COMMENTED** [review]
🫡
